cuenca/http/client.py

```diff
@@ -104,6 +105,12 @@ def request(
             if self.jwt_token.is_expired:
                 self.jwt_token = Jwt.create(self)
             self.session.headers['X-Cuenca-Token'] = self.jwt_token.token
+
+        if data is not None:
+            for key, value in data.items():
+                if isinstance(value, (dt.date, dt.datetime)):
+                    data[key] = value.isoformat()
```

Comment:
missing the timezone:
```python
In [28]: value = dt.datetime.now()

In [29]: value.isoformat()
Out[29]: '2021-08-13T17:49:16.368700'

In [30]: value.tzinfo is None
Out[30]: True

In [31]: default_tz = dt.timezone.utc

In [32]: if value.tzinfo is None:
    ...:     value = value.astimezone(default_tz)
    ...:

In [33]: value.isoformat()
Out[33]: '2021-08-13T22:49:16.368700+00:00'
```

not sure what `default_tz` would be in this case: UTC or America/Mexico_City

Reply:
Si es necesario indicar el timezone?
Entiendo que todo se maneja en UTC